src/js/big-play-button.js

```diff
@@ -44,9 +56,23 @@ class BigPlayButton extends Button {
       return;
     }
 
-    this.setTimeout(function() {
-      playToggle.focus();
-    }, 1);
+    if (playPromise) {
```

Comment:
Should this duck-type the play promise (e.g. `if (playPromise && playPromise.then) {`)?